pkg/k8s/watchers/pod.go

```diff
@@ -155,6 +155,14 @@ type K8sPodWatcher struct {
 	controllersStarted chan struct{}
 }
 
+func hostPortServiceName(pod *slim_corev1.Pod, port uint16, cluster string) *loadbalancer.ServiceName {
+	return &loadbalancer.ServiceName{
+		Name:      fmt.Sprintf("%s/pod-id/%s/host-port/%d", pod.ObjectMeta.Name, pod.ObjectMeta.UID, port),
```

Comment:
Let's use string concat directly here to avoid `Sprintf` in the hotpath code.

Reply:
I'm not expert enough in golang to make the suggested change on my own such that there's an actual performance improvement.  I think you're asking for something like this:
```go
pod.ObjectMeta.Name + "/pod-id/" + pod.ObjectMeta.UID + "/host-port/" + strconv.Itoa(port)
```
If that's not right, can you provide an example of what you want done here?

Reply:
@christarazi, can you please get back to me soon?

Reply:
Yes, that's correct. Thanks!

Reply:
The changes are in, but they were a little more complicated than my suggestion as it turns out.  Please check again and see if they're sufficient.